source/boulder/stages/chroot_package.d

```diff
@@ -0,0 +1,92 @@
+/*
+ * SPDX-FileCopyrightText: Copyright © 2020-2023 Serpent OS Developers
+ *
+ * SPDX-License-Identifier: Zlib
+ */
+
+/**
+ * boulder.stages.chroot_package
+ *
+ * Implements chroot'ing into a package
```

Comment:
Nit: Are we chrooting into a package? Aren't we chrooting into a package recipe build environment?

For context, I consider "a package" the finished article. While it's in progress, it feels like "a package recipe build operation" to me?